apps/src/code-studio/pd/application/facilitator1920/Section2ChooseYourProgram.jsx

```diff
@@ -4,63 +4,176 @@ import LabeledFormComponent from "../../form_components/LabeledFormComponent";
+              answers: [
+                {
+                  answerText: 'CS Fundamentals (K - 5th grade)',
+                  answerValue: CSF
+                },{
+                  answerText: 'CS Discoveries (6 - 10th grade)',
+                  answerValue: CSD
+                },{
+                  answerText: 'CS Principles (9 - 12th grade)',
+                  answerValue: CSP
+                }
+              ],
+              controlWidth: {md: 6},
+              onChange: this.onProgramSelect.bind(this),
             }
-          </div>
+          )
         }
 
-        {this.props.data.program && this.props.data.program !== PROGRAM_CSF &&
+        {
+          this.props.data.program === CSF &&
           <div>
-            <h4>
-              If selected for the program, you will be required to attend TeacherCon and a
-              Facilitator-in-Training Weekend in the summer of 2019.
-            </h4>
+            <p>
+              Code.org facilitators work with their assigned Regional Partner to host workshops
+              for teachers in their region. Facilitator applicants are assigned to Regional
+              Partners based on the zip code they provide in their application.
+            </p>
+
+            {
+              !this.props.data.regionalPartnerId &&
+              <div>
+                <p>
+                  <strong>There is no Regional Partner supporting CS Fundamentals in your region at this time.</strong>
+                </p>
+                <p>
+                  Please note that we prioritize applicants in regions where we currently have a
+                  Regional Partner supporting CS Fundamentals, and there is a need for additional
+                  facilitators. Code.org will review your application and contact you if there is
+                  a need for facilitators. We are not able to guarantee a space for you in a
+                  different location.
+                </p>
+              </div>
+            }
+            {
+              this.props.data.regionalPartnerId && PartnersWithoutCsf.includes(this.props.data.regionalPartnerId) &&
+              <div>
+                <p>
+                  <strong>Your Regional Partner is not accepting applications for CS Fundamentals facilitators at this time.</strong>
+                </p>
+                <p>
+                  Please note that we prioritize applicants in regions where we currently have a
+                  Regional Partner supporting CS Fundamentals, and there is a need for additional
+                  facilitators. Code.org will review your application and contact you if there is
+                  a need for facilitators. We are not able to guarantee a space for you in a
+                  different location.
+                </p>
+              </div>
+            }
+            {
+              this.props.data.regionalPartnerId && !PartnersWithoutCsf.includes(this.props.data.regionalPartnerId) &&
```

Comment:
The way that we've implemented PartnersWithoutCsf is dodgy - we really shouldn't have hardcoded IDs anywhere. Instead, this would have been better implemented as an attribute on the regional partner.

We'll have to take this as a tech debt work item